rsmtool/notebooks/comparison/score_distributions.ipynb

```diff
@@ -8,6 +8,41 @@
    ]
   },
   {
+   "cell_type": "code",
```

Comment:
I agree. What I meant is that it would be in the comparison report, but as a separate notebook consistency.ipynb, in the same way as it is handled in rsmtool. This way the user will have control over whether the notebook is in the final report or not. So essentially you would take this code and copy it into a new notebook. It would also require minor changes to report.py (adding the new notebook to the section list).